components/org.wso2.ppaas.manager.console/console/configure_form.jag

```diff
@@ -101,8 +101,8 @@ if(isEdit == false && elements.action != 'new') {
             break;
 
         case "autoscaling-policies":
-            formTitle = "Auto-scaling Policie",
-                    buttonText = "Auto-scaling Policy";
+            formTitle = "Autoscaling Policie",
```

Comment:
May be this title is used in Add Autoscaling Policy page. Then it should be Autoscaling Policy.